cg/services/analysis_starter/configurator/file_creators/utils.py

```diff
@@ -0,0 +1,51 @@
+import logging
```

Comment:
This is the most important module! The functions for creating the Path, creating the content, and writing the content are defined here without belonging to any pipeline class